Makefile.common.mk

```diff
@@ -22,19 +22,19 @@
 IMG = docker.io/sdake/build-tools:2019-08-03
 UID = $(shell id -u)
 PWD = $(shell pwd)
-GOBIN ?= $(GOPATH)/bin
+GOBIN_SOURCE ?= $(GOPATH)/bin
+GOBIN ?= /work/out/bin
 
 RUN = docker run -t --sig-proxy=true -u $(UID) --rm \
 	-v /etc/passwd:/etc/passwd:ro \
-	-v /etc/passwd:/etc/passwd:ro \
-	-v /etc/localtime:/etc/localtime:ro \
-	-v /etc/timezeone:/etc/timezeone:ro \
+	-v $(reallink /etc/localtime):/etc/localtime:ro \
```

Comment:
Should this have been readlink and not reallink?

Reply:
`readlink` is not available on mac without brew.  `reallink` is available on both Mac and Linux.

Goal of the build system is make + docker and you are GTG.

Cheers
-steve

Reply:
Interesting as I don't have `reallink` on my Mac, but had installed `readlink` earlier.